.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: false
+language: go
+
+go:
+    - 1.7
+
+env:
+    global:
+        - GO15VENDOREXPERIMENT=1
```

Comment:
since we only use 1.7 we can drop this (it's enabled by default in 1.6 and always on in 1.7)